### PR TITLE
Update description of SENT-template

### PR DESCRIPTION
**Pull request by Zimtir**
**https://github.com/Zimtir/SENT-template**
**Description of SENT-template not contain enough information about provided features like in others tools**

<br>

By submitting this pull request, I promise I have read the [contribution guidelines](/../contributing.md) twice and ensured my submission follows it.

### CODE REVIEW

**Comment by antony** [comment]
@Zimtir I'm not sure this is the best place to enumerate your entire support graph. Can you limit the description to maybe the top 5 you want to list and then "and many more" or similar please?

**Comment by Zimtir** [comment]
@antony looks better, doesn't it?

File: readme.md [hunk 1]
```diff
@@ -46,7 +46,7 @@
 - [sapper-typescript-graphql-template](https://github.com/babichjacob/sapper-typescript-graphql-template)
   - Template that includes Sapper, TypeScript preprocessing, and a GraphQL server through TypeGraphQL.
 - [SENT-template](https://github.com/Zimtir/SENT-template)
-  - SENT (Sapper Express Node Template) and other tools.
+  - A small subframework with tools for full development flow like Tests, Docker, Linters, Component Library, Security, TypeScript and other tools)
```
Comment (benmccann) [inline readme.md:49]: There's an ending paren without a matching start paren and most of these words don't need to be capitalized. How about this:

```suggestion
  - A small sub-framework with tools for full development flow like tests, Docker, linting, component library, security, TypeScript, etc.
```

**Comment by Zimtir** [comment]
@antony can it be merged?

**Comment by Zimtir** [comment]
@benmccann PTAL when you will have a time

**Comment by Zimtir** [comment]
@antony could you merge it?

**Comment by Zimtir** [comment]
@benmccann @antony can it be merged?

**Review by antony — APPROVED** [review]

**Pull request merged**